### DISCUSSION TITLE
Phân công tuần 11

### DISCUSSION THREAD

**Discussion by baonhu1304** [baonhu1304/Lop-10A2-nhom-1#24] [Announcements]
Câu 1 bài 9 mỗi bạn đều làm nha. Còn 4 câu mõi đứa 1 câu nhé. Ai chọn sau trúng câu khó gáng chịu à nghen.

**Comment by thaopham2501** [comment]
ủa là sao :>>
tại sai câu 1 bài 9 đều làm

**Reply by letu25082006** [reply]
tại câu đó là câu thực tế nên mỗi ng mỗi khác suy ra ai cũng làm

**Reply by thaopham2501** [reply]
u là chòi :))) làm khó nhau dzãy thầy

**Comment by letu25082006** [comment]
mtu câu mụt bài ten

**Reply by baonhu1304** [reply]
https://www.thegioididong.com/hoi-dap/he-dieu-hanh-la-gi-804907
đợt trc tui làm có kiếm đc cái này nè bà

**Comment by thaopham2501** [comment]
ổ kê câu 3 bài 9 :))

**Comment by baonhu1304** [comment]
Tui câu 2 bài 10 nhé

**Comment by Lenga10A2** [comment]
ủa v tui câu 2 bài 9 hả :)) ố mai gót dài phết nhở

**Reply by baonhu1304** [reply]
Câu này Nga với @NguyenNgocNhuY cùng làm nha

**Reply by letu25082006** [reply]
khum xao chị em sp

**Reply by Lenga10A2** [reply]
oki cảm mơn các chị em :)) v tui vs Ý làm chung hửm

**Comment by letu25082006** [comment]
dl ngày mấy zị @baonhu1304

**Reply by baonhu1304** [reply]
28/11 lận nghen. Làm sớm nghỉ sớm ii. Tuần sau còn ôn bài ktra nựa

**Reply by letu25082006** [reply]
okii

**Comment by baonhu1304** [comment]
Câu 1 bài 9 tui tạo 1 cái Column chung. Ai viết ý kiến nhớ ghi chú tên mình trên cùng nha

**Reply by letu25082006** [reply]
ấu kii

**Comment by Lenga10A2** [comment]
chu choa giờ đọc câu 2 bài 9 xong ko hỉu j lun á tr :((

**Reply by baonhu1304** [reply]
Từ từ bìn tĩn

**Comment by NguyenNgocNhuY** [comment]
@Lenga10A2 v mình chia phần để trả lời đi bà

**Reply by Lenga10A2** [reply]
Ừm v bà ý đầu ha còn phần cảm nghĩ tui làm cho còn 5 vd thì mình làm chung ok ko bà
